src/Calendar.js

```diff
@@ -898,6 +898,8 @@ class Calendar extends React.Component {
     resourceIdAccessor: 'id',
     resourceTitleAccessor: 'title',
 
+    eventIdAccessor: 'id',
```

Comment:
One last bit is to add `eventIdAccessor` to the `propTypes` at the top of the file.

Reply:
done! thanks